icicle-source/src/Icicle/Source/Checker/Invariants.hs

```diff
@@ -77,12 +74,12 @@ invariantQ ctx (Query (c:cs) xfinal)
 
   goBanAll
      = invariantQ
-        (ctx{ checkInvariants = inv{ allowLatest = False, allowWindowsOrGroups = False }})
+        (ctx{ checkInvariants = inv{ allowLatest = False, allowWindowsOrGroups = True }})
         q'
 
   goBanGroup
      = invariantQ
-        (ctx{ checkInvariants = inv{ allowWindowsOrGroups = False }})
+        (ctx{ checkInvariants = inv{ allowWindowsOrGroups = True }})
```

Comment:
is `allowWindowsOrGroups` still necessary then? maybe these should remain `False` and just rename it to `allowWindows`